### PR TITLE
Fix AzDO area path for policy violations reporting

### PR DESCRIPTION
**Pull request by mkArtakMSFT**

### CODE REVIEW

File: eng/sdl-tsa-vars.config [hunk 1]
```diff
@@ -3,7 +3,7 @@
 -TsaProjectName DEVDIV
 -TsaNotificationEmail aspnetcore-build@microsoft.com
 -TsaCodebaseAdmin REDMOND\kevinpi
--TsaBugAreaPath "DevDiv\ASP.NET Core"
```
Comment (mkArtakMSFT) [inline eng/sdl-tsa-vars.config:6]: @dougbu, @wtgodbe I've noticed that there is a difference between `\` and `\\` in these two files. I'd like to just use the same `\\` in both files. Do you know if that may cause any issues?

Comment (wtgodbe) [reply eng/sdl-tsa-vars.config:6]: @dotnet/dnceng would know for sure, but I suspect `\\` here will be literal, which may cause problems with area assignment

**Review by wtgodbe — APPROVED** [review]

File: eng/sdl-tsa-vars.config (hunk 1 above)
Comment (michellemcdaniel) [reply eng/sdl-tsa-vars.config:6]: I'm not sure what is using the tsaoptions.json, but the two files are read by two different processes which probably interpret string literals differently. The sdl-tsa-vars.config file is read by the release pipeline as a text file and doesn't need the slashes to be escaped, where the tsaoptions.json file probably does require escaping.

Comment (garath) [reply eng/sdl-tsa-vars.config:6]: Yes, I believe that's correct. Having two slashes here will produce two slashes in the TSA configuration file, and I'm not sure that TSA will handle it properly.

Comment (mkArtakMSFT) [reply eng/sdl-tsa-vars.config:6]: Good that I've checked then 👍 
I'll leave this as is.

**Pull request merged**